### PR TITLE
Update README.md, by adding some sections and placeholders

### PR DESCRIPTION
**Pull request by moodmosaic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,46 @@
...
+
+TODO
+
+## NuGet
+
+There isn't much here yet, but `dotnet-jack` can be published on NuGet once it's in a usable state.
+
+## Versioning
+
+`dotnet-jack` follows [Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html) once it’s in a usable state.
```
Comment (moodmosaic) [inline README.md]: I've mentioned [SemVer](http://semver.org/spec/v2.0.0.html) here, but if you disagree I'll remove it :smiley:

**Comment by moodmosaic** [comment]
A preview of how it looks like is shown [here](https://github.com/moodmosaic/dotnet-jack/tree/add-sections-and-placeholders-to-the-readme#dotnet-jack).

**Comment by moodmosaic** [comment]
Another idea is to port [the original QuickCheck manual](http://www.cse.chalmers.se/~rjmh/QuickCheck/manual.html) in Jack, and put it on the README.md, highlighting the areas where Jack is different.

**Comment by moodmosaic** [comment]
FWIW, I've pushed a commit to this (boring) pull request. – You may preview the rendered markdown [here](https://github.com/moodmosaic/dotnet-jack/blob/add-sections-and-placeholders-to-the-readme/README.md).

**Comment by jystic** [comment]
This looks good 👍

**Pull request merged**